pulp_2to3_migration/app/plugin/deb/repository.py

```diff
@@ -78,7 +78,7 @@ def migrate_to_pulp3(cls, pulp2distributor, repo_version):
         pulp2_config = pulp2distributor.pulp2_config
         distribution_data = cls.parse_base_config(pulp2distributor, pulp2_config)
         base_path = pulp2_config.get('relative_url', pulp2distributor.pulp2_repo_id)
-        distribution_data['base_path'] = base_path.rstrip('/')
+        distribution_data['base_path'] = base_path.strip('/')
```

Comment:
In effect this minor code change removes leading slashes from the `base_path` during migration.
My intuition is that leading slashes on the `base_path` do not make a difference either way, but that it is cleaner not to have them, so I am cautiously in favor of the code change.

Reply:
Do you need this to be backported to 0.11?
This change doesn't seem to relate to the PR title, it's a code change.
I can merge it as a part of test PR if not backport is needed. If you want to backport it to 0.11, please file an issue.
Actually if you prefer tests to be backported, please file an issue for them as well.

Reply:
As I understand it it was the simplest/only way to make all the tests turn green, and it makes sense as a non-critical code improvement (since leading slashes on a `base_path` should not hurt, but aren't meaningful either). The reason I originally used `rstrip` was copy and paste from other content types. It might make sense to backport the tests (along with this change) to 0.11, but it probably makes most sense to open a PR for that once this one has been merged (rather than using the usual cherry-pick process)?

Reply:
Both ways work for me.
I would cherry-pick anything needed and that's it but if you prefer to open separate PRs for each branch, feel free to do so.
It won't be acceptable for any change but because the change is small and the rest are tests, we can do your preferred way this time.

Reply:
I opened a Backport-ticket for the code change (https://pulp.plan.io/issues/9188)
For the tests, we will open a separate PR so we can react to failing tests (if they occur).